cmd/keeper/keeper.go

```diff
@@ -1754,3 +1757,45 @@ func keeper(c *cobra.Command, args []string) {
 
 	lockFile.Close()
 }
+
+func (p *PostgresKeeper) generateHBA(cd *cluster.ClusterData, role common.Role, userHba []string) []string {
+
+	// Dynamically generate hba auth from clusterData
+	var hbaBase []string
+
+	switch *cd.Cluster.DefSpec().DefaultSUReplAccessMode {
+	case cluster.SUReplAccessAll:
+		hbaBase = append(
+			hbaBase,
+			fmt.Sprintf("host all %s %s %s\n", p.pgSUUsername, "0.0.0.0/0", p.pgSUAuthMethod),
+			fmt.Sprintf("host all %s %s %s\n", p.pgSUUsername, "::0/0", p.pgSUAuthMethod),
+			fmt.Sprintf("host replication %s %s %s\n", p.pgReplUsername, "0.0.0.0/0", p.pgReplAuthMethod),
+			fmt.Sprintf("host replication %s %s %s\n", p.pgReplUsername, "::0/0", p.pgReplAuthMethod),
+		)
+	case cluster.SUReplAccessStrict:
+		// allow for all servers su access
+		hbaBase = append(
+			hbaBase,
+			fmt.Sprintf("host all %s %s %s\n", p.pgSUUsername, "127.0.0.1/32", p.pgReplAuthMethod),
```

Comment:
This is only from 127.0.0.1/32 not from all the standbys or am I missing something?